### ISSUE TITLE
Latest PDF theme fails

### ISSUE THREAD

**Issue by sameo** [riscv/docs-resources#13]
When using the latest docs-resources code (005cb0db1c46c31fff163914f3e480dbda0306be), I get the following error:

```
❯ make build-no-container
Starting build...
make SKIP_DOCKER=true build-docs
make[1]: Entering directory '/home/samuel/devlp/specs/riscv/riscv-ap-tee-io'
asciidoctor-pdf --trace -a compress -a mathematical-format=svg -a revnumber=v0.0.0 -a revremark=Draft -a revdate=2024-04-30 -a pdf-fontsdir=docs-resources/fonts -a pdf-theme=docs-resources/themes/riscv-pdf.yml  -D build --failure-level=ERROR --require=asciidoctor-bibtex --require=asciidoctor-diagram --require=asciidoctor-mathematical src/riscv-cove-io.adoc
asciidoctor: ERROR: could not locate or load the built-in pdf theme `docs-resources/themes/riscv-pdf.yml' because of NoMethodError undefined method `each' for "646b74":String; reverting to default theme
make[1]: *** [Makefile:61: riscv-cove-io.pdf] Error 1
make[1]: Leaving directory '/home/samuel/devlp/specs/riscv/riscv-ap-tee-io'
make: *** [Makefile:83: build-no-container] Error 2
```

**Comment by sameo** [comment]
Using commit `37ec11de071c846b687386b4315000c7fd8e3195` does **not** show this error.

**Comment by wmat** [comment]
Did you populate the submodule?  

git submodule update --init

**Comment by wmat** [comment]
Got it.  There's a bug in the riscv-pdf.yml file.  Working on it now.

**Comment by sameo** [comment]
I think this is fixed with #14 ?

**Comment by wmat** [comment]
@sameo closing this one as the riscv-ap-tee-io repo is building with the latest version of docs-resources.  Also, make build-no-container works locally.
